client/notifications/notifications.tsx

```diff
@@ -3,12 +3,16 @@ import styled from 'styled-components'
 import { colorDividers } from '../styles/colors'
 import { body1 } from '../styles/typography'
 
-const Container = styled.div<{ unread: boolean }>`
+const Container = styled.div<{ read: boolean }>`
```

Comment:
I probably added this so it's my mistake, but styled-components has a shorthand for props that should not be output to the DOM element now, which you should try to use (just prepend any such prop with a `$`)